.github/workflows/boulder-ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       matrix:
         # Add additional docker image tags here and all tests will be run with the additional image.
         BOULDER_TOOLS_TAG:
-          - go1.21.1_2023-09-07
+          - go1.21.3_2023-10-12
```

Comment:
This matrix should list both versions until we don't need to test on go1.21.1 anymore.